src/Compilers/CSharp/Test/Symbol/Symbols/Source/BaseClassTests.cs

```diff
@@ -2301,5 +2301,41 @@ class F : A<D*>.B { }
                 //     class E : A<C*>.B { }
                 Diagnostic(ErrorCode.ERR_ManagedAddr, "E").WithArguments("Base.C").WithLocation(12, 11));
         }
+
+        [Fact]
+        [WorkItem(1107185, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1107185")]
+        public void Tuple_MissingNestedTypeArgument_01()
+        {
+            var source =
+@"interface I<T>
+{
+}
+class A : I<(object, A.B)>
```

Comment:
>(object, A.B) [](start = 12, length = 13)

Are we going to get the same behavior when this is explicit ```ValueTuple<object, A.B>```?

Reply:
Yes. Added tests.

---
In reply to: [423252574](https://github.com/dotnet/roslyn/pull/44098#discussion_r423252574) [](ancestors = 423252574)